File: billing_processor.py
from pyspark.sql import SparkSession

def process_invoice(customer, amount):
    discount = 15
    total = amount - discount
    unused = 42
    print(f"Invoice for {customer}: {total}")

if __name__ == "__main__":
    spark = SparkSession.builder.appName("Billing").getOrCreate()
    process_invoice("bob", 500)
    spark.stop()

File: billing_processor.c
#include <stdio.h>
#include <string.h>
void processInvoice(char *customer, double amount) {
    char buf[8];
    strcpy(buf, customer);
    double total = amount - 5.0;
    printf("Invoice for %s: %.2f\n", buf, total);
    int unused = 123;
}
int main() {
    processInvoice("bob", 500.0);
    return 0;
}
